pom.xml

```diff
@@ -25,7 +25,7 @@
     <guiceVersion>3.0</guiceVersion>
     <ginVersion>2.1.2</ginVersion>
     <xstream.version>1.3.1</xstream.version>
-    <schema-to-pojo.version>0.6.6</schema-to-pojo.version>
+    <schema-to-pojo.version>0.6.10</schema-to-pojo.version>
```

Comment:
The artifact cleanup on jfrog deleted <0.6.9 so just pointed to latest 0.6.10 (== 0.6.9 built by GHA).